src/Message/Message.tsx

```diff
@@ -29,22 +22,21 @@ interface IWrapper {
 }
 
 const Wrapper = styled.div<IWrapper>`
-  font-family: ${theme.font.system};
   align-items: center;
   border: 2px solid
-    ${({ type }): string =>
-      type === 'warning' ? `${theme.colors.red7}` : `${theme.colors.blue7}`};
+    ${({ type }) =>
+      type === 'warning' ? theme.colors.red7 : theme.colors.blue7};
   box-sizing: border-box;
   border-radius: 8px;
   margin-bottom: 24px;
   padding: 16px;
   display: flex;
+  font-family: ${theme.font.system};
+  color: ${({ type }) =>
```

Comment:
might be worth destructuring `({ type })` once 
```js
styled.div<IWrapper>(({ type }) => css`...`);
```

Reply:
Done!